torchvision/transforms/transforms.py

```diff
@@ -570,7 +576,7 @@ def get_params(img, scale, ratio):
             w = int(round(math.sqrt(target_area * aspect_ratio)))
             h = int(round(math.sqrt(target_area / aspect_ratio)))
 
-            if random.random() < 0.5:
+            if random.random() < 0.5 and min(ratio) <= (w / h) <= max(ratio):
```

Comment:
For python2 compatibility
```suggestion
            if random.random() < 0.5 and min(ratio) <= (float(w) / h) <= max(ratio):
```